pkg/controller/ttlafterfinished/ttlafterfinished_controller.go

```diff
@@ -131,15 +131,15 @@ func (tc *Controller) addJob(obj interface{}) {
 
 func (tc *Controller) updateJob(old, cur interface{}) {
 	job := cur.(*batch.Job)
-	klog.V(4).Infof("Updating job %s/%s", job.Namespace, job.Name)
+	klog.V(4).InfoS("Updating Job", "jobNamespace", job.Namespace, "job", job.Name)
```

Comment:
Same as above

Reply:
This can actually use `klog.KRef`, thanks for catching it.